lib/screens/downloads_settings_screen.dart

```diff
@@ -37,52 +37,51 @@ class DownloadsSettingsScreen extends StatelessWidget {
           ),
           if (Platform.isIOS || Platform.isAndroid) const RequireWifiSwitch(),
           const ShowPlaylistSongsSwitch(),
+          const SyncFavoritesSwitch(),
+          ListTile(
+            // TODO real UI for this
+            title: const Text("Show all playlists offline"),
+            subtitle: const Text(
+                "Sync metadata for all playlists to show partially downloaded playlists offline"),
```

Comment:
Is there a particular reason you didn't move this to the localizations yet, or did you just miss it?

Reply:
None of the DownloadButton based settings have been moved to localizations yet.  I should probably do that.